pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -569,27 +568,46 @@ func getFirstContainerWithSourceVolume(containers []corev1.Container) (string, s
 }
 
 // Delete deletes the component
-func (a Adapter) Delete(labels map[string]string) error {
-	spinner := log.Spinnerf("Deleting devfile component %s", a.ComponentName)
-	defer spinner.End(false)
+func (a Adapter) Delete(labels map[string]string, show bool) error {
 
-	componentExists, err := utils.ComponentExists(a.Client, a.ComponentName)
+	log.Infof("\nGathering information for component %s", a.ComponentName)
+	podSpinner := log.Spinnerf("Retrieving pod for component %s", a.ComponentName)
```

Comment:
We shouldn't display terms like pod in the UI. This should be something simpler like `Checking component status` or `Retrieving component info`

Reply:
I am going to say `Retrieving info for component %s`

Reply:
updated